src/main/java/module-info.java

```diff
@@ -0,0 +1,57 @@
+module graphics.scenery {
+    exports graphics.scenery;
+    exports graphics.scenery.atomicsimulations;
+    exports graphics.scenery.attribute;
+    exports graphics.scenery.backends;
+    exports graphics.scenery.compute;
+    exports graphics.scenery.controls;
+    exports graphics.scenery.effectors;
+    exports graphics.scenery.fonts;
+    exports graphics.scenery.geometry;
+    exports graphics.scenery.net;
+    exports graphics.scenery.numerics;
+    exports graphics.scenery.primitives;
+    exports graphics.scenery.proteins;
+    exports graphics.scenery.repl;
+    exports graphics.scenery.serialization;
+    exports graphics.scenery.textures;
+    exports graphics.scenery.ui;
+    exports graphics.scenery.utils;
+    exports graphics.scenery.volumes;
```

Comment:
If I'm reading it right, it looks like this is exporting *all* the `graphics.scenery.*` packages, but none of the `graphics.scenery.*.*` ones? Specifically: missing are `attribute.geometry`, `attribute.material`, `attribute.renderable`, `attribute.spatial`, `backends.software`, `backends.vulkan`, `controls.behaviours`, `controls.eyetracking`, `utils.extensions`. Intentional, or no?

And are you sure you want to expose so much API surface? I guess since scenery hasn't hit 1.0.0 yet, you can still break things. But this work is an opportunity to consider whether it makes sense to create any internal (non-exported) packages, no?